api/src/main/java/org/openmrs/module/pihcore/setup/HtmlFormSetup.java

```diff
@@ -20,4 +29,26 @@ public static void setupHtmlFormEntryTagHandlers() throws Exception {
         htmlFormEntryService.addHandler(PihCoreConstants.HTMLFORMENTRY_FAMILY_HISTORY_RELATIVE_CHECKBOXES_TAG_NAME, new FamilyHistoryRelativeCheckboxesTagHandler());
     }
 
+    public static void loadHtmlForms() {
+
+        ResourceFactory resourceFactory = ResourceFactory.getInstance();
+        FormService formService = Context.getFormService();
+        HtmlFormEntryService htmlFormEntryService = Context.getService(HtmlFormEntryService.class);
+
+        try {
+            File htmlformDir = new File(PihCoreUtil.getFormDirectory());
+            Collection<File> files = FileUtils.listFiles(htmlformDir, null, true);
+
+            for (File file : files) {
+                try {
+                    HtmlFormUtil.getHtmlFormFromUiResource(resourceFactory, formService, htmlFormEntryService, "file:" + file.getAbsolutePath());
+                } catch (IOException e) {
+                    log.error("Unable to load HTML Form at path: " + file, e);
+                }
+            }
+        }
+        catch (Exception e) {
+            log.error("Unable to load HTML forms--this error is expected when running component tests");
+        }
+    }
 }
```

Comment:
Given that this is part of the PR, maybe this is about more than just loading concepts in the background?

Reply:
True... I was struggling with the name of the task, for what it's worth.  MetadataSetupTask seemed to vague, especially when it doesn't set up *all* metadata, but ConceptsAndDrugsAndFormsAndProgramsMetadataSetupTask didn't sound good as well.  In the end I went with MetadataSetupTask with the hopes that this will all be refactored within the year anyway.